File: app/src/test/java/com/example/pie/android/model/TodoItemTest.java
package com.example.pie.android.model;

import org.junit.Before;
import org.junit.Test;

import static junit.framework.Assert.*;

public class TodoItemTest {

    private TodoItem testObj;

    @Before
    public void setUp() {
        testObj = new TodoItem();
    }

    @Test
    public void testGettersAndSetters() {
        int id = 23;
        String task = "Do android testing";
        boolean isDone = false;

        testObj.setId(id);
        testObj.setTask(task);
        testObj.setDone(isDone);

        assertEquals(isDone, testObj.getDone());
        assertEquals(id, testObj.getId());
        assertEquals(task, testObj.getTask());
    }
}

File: app/src/test/java/com/example/pie/android/model/TodoItemTest.kt
package com.example.pie.android.model

import org.junit.Before
import org.junit.Test

import junit.framework.Assert.*

class TodoItemTest {

    private lateinit var testObj: TodoItem

    @Before
    fun setUp() {
        testObj = TodoItem()
    }

    @Test
    fun testGettersAndSetters() {
        val id = 23
        val task = "Do android testing"
        val isDone = false

        testObj.id = id
        testObj.task = task
        testObj.done = isDone

        assertEquals(isDone, testObj.done)
        assertEquals(id, testObj.id)
        assertEquals(task, testObj.task)
    }
}
